src/translation/TranslatableStrings.as

```diff
@@ -128,6 +128,7 @@ public class TranslatableStrings {
 		addAll(VariableSettings.strings());
 		addAll(Watcher.strings());
 		addAll(CameraDialog.strings());
+		addAll(PrivacyDialog.strings());
```

Comment:
It looks like you might have forgotten to add `PrivacyDialog.as` to this branch. That said, if possible I think it would be a good idea to add it in `scratch-flash-offline` instead since it won't be used by the online editor or by ScratchX.

Reply:
@cwillisf PrivacyDialog.as should be included in `scratch-flash-offline`. https://github.com/LLK/scratch-flash-offline/blob/offline-editor-telemetry/src/PrivacyDialog.as

Reply:
Ah, I see. In that case the call to `PrivacyDialog.strings()` will need to be in `scratch-flash-offline` as well: as it is, this will cause a build error for the online editor and ScratchX.